cli/pkg/local/app.go

```diff
@@ -332,13 +336,9 @@ func (a *App) pollServer(ctx context.Context, httpPort int, openOnHealthy bool)
 			return
 		}
 
-		// Check if server is up
-		resp, err := client.Get(uri + "/v1/ping")
-		if err == nil {
-			defer resp.Body.Close()
-			if resp.StatusCode < http.StatusInternalServerError {
-				break
-			}
+		ok := serverHealthCheck(client, uri)
+		if !ok {
+			break
```

Comment:
Actually, this shouldn't be a separate function. The `defer` is not needed here, you can just do `resp.Body.Close()` when `err == nil`